From: "J. R. Okajima" <hooanon05@yahoo.co.jp>
Date: Wed, 18 Jan 2012 11:45:10 +0900
Subject: [08/83] aufs: tiny for 3.3, arg of ->migratepage()
Origin: http://sourceforge.net/p/aufs/aufs3-standalone/ci/f5311566b3c9ce2eeda5df5422f8d2b3dcdb2d48

Signed-off-by: J. R. Okajima <hooanon05@yahoo.co.jp>
(cherry picked from commit 1d73cedce0be6fa86cd5e0ee39af73a123478a0f)
---
 fs/aufs/file.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/fs/aufs/file.c b/fs/aufs/file.c
index 0bc2eca..120d61f 100644
--- a/fs/aufs/file.c
+++ b/fs/aufs/file.c
@@ -20,6 +20,9 @@
  * handling file/dir, and address_space operation
  */
 
+#ifdef CONFIG_AUFS_DEBUG
+#include <linux/migrate.h>
+#endif
 #include <linux/pagemap.h>
 #include "aufs.h"
 
@@ -638,7 +641,7 @@ static void aufs_invalidatepage(struct page *page, unsigned long offset)
 static int aufs_releasepage(struct page *page, gfp_t gfp)
 { AuUnsupport(); return 0; }
 static int aufs_migratepage(struct address_space *mapping, struct page *newpage,
-			    struct page *page)
+			    struct page *page, enum migrate_mode mode)
 { AuUnsupport(); return 0; }
 static int aufs_launder_page(struct page *page)
 { AuUnsupport(); return 0; }
